modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -35,6 +54,13 @@ function getModuleFields($module, $view='EditView',$value = '', $valid = array()
             $mod = new $beanList[$module]();
             foreach($mod->field_defs as $name => $arr){
                 if(ACLController::checkAccess($mod->module_dir, 'list', true)) {
+
+                    if (array_key_exists($mod->module_dir, $blockedModuleFields)) {
+                        if(in_array($arr['name'], $blockedModuleFields[$mod->module_dir]) && !$current_user->isAdmin()) {
+                            continue;
```

Comment:
Can we also add a log note or info or something at least, to let developers and/or users know why the workflow (or report) doesn't works for these fields now...